.github/actions/setup-node-ci/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Setup Node for REST API CI
+description: Uses specified Node version and runs npm commands to set up the environment for REST API CI
+
+inputs:
+  node-version:
+    description: 'Node version to use'
+    default: 20.x
+    
+
+runs:
+  using: "composite"
+  steps:
+    - name: Use Node ${{ inputs.node-version }}
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ inputs.node-version }}
+
+    - run: npm ci
+      shell: pwsh
```

Comment:
```suggestion
```

I recommend we simplify and use the default shell, unless we need pwsh

Reply:
![image](https://github.com/user-attachments/assets/287c9fe2-35e6-4323-b42f-58f528198103)

`shell` is required in composite actions, it seems. `pwsh` and `bash` are available on all agents.

Reply:
Let's try `bash`